src/pip/_internal/vcs/__init__.py

```diff
@@ -422,7 +414,7 @@ def get_url(self, location):
         """
         Return the url used at location
 
-        This is used in get_info() and obtain().
+        This is used in obtain().
```

Comment:
I would add `get_src_requirement()`.

Reply:
We can just remove the "used in" comment IMO. It's easy enough to understand what this method is for without having to mention which other methods use it.

(that can't be said about some other methods)